test-status/db.js

```diff
@@ -83,3 +83,15 @@ exports.getCheckRunResults = async (db, headSha, type, subType) => {
     return existingCheck;
   }
 };
+
+/**
+ * Get the GitHub username of the current build cop.
+ *
+ * @param {!Knex} db instantiated database connection.
+ * @return {string} the GitHub username of the current build cop.
+ */
+exports.getBuildCop = async db => {
+  return await db('buildCop')
+      .pluck('username')
```

Comment:
nit: `first('username').then(username => username)`

Reply:
Done. It's `first('username').then(row => row['username'])` but that's still more readable than using `pluck` :)